### PR TITLE
Instance api improvments

### PR DESCRIPTION
**Pull request by cp16net**
This adds some links and flavors and goodies to the list instance(s).
As well as add some exception handling from the service.

### CODE REVIEW

**Comment by hub-cap** [comment]
Tests? :)

File: reddwarf/database/service.py [hunk 1]
```diff
@@ -23,20 +23,53 @@
...
 from reddwarf.common import context as rd_context
 from reddwarf.common import exception
+from reddwarf.common import utils
 from reddwarf.common import wsgi
 from reddwarf.database import models
 from reddwarf.database import views
 
 CONFIG = config.Config
-LOG = logging.getLogger('reddwarf.database.service')
+LOG = logging.getLogger(__name__)
```
Comment (ed-) [inline reddwarf/database/service.py:32]: ++

File: reddwarf/database/service.py [hunk 2]
```diff
@@ -23,20 +23,53 @@
...
 
 
 class BaseController(wsgi.Controller):
     """Base controller class."""
 
+    exclude_attr = []
+    exception_map = {
```
Comment (ed-) [inline reddwarf/database/service.py:39]: I like this

**Comment by ed-** [comment]
+(tests?1:0)

**Comment by hub-cap** [comment]
So once you are done can u squash the commits into a single commit msg like this.

"Summary of commit, such as 'adding additional functionality to X
(a BLANK line)
- Detailed information about Thing 1
- Detailed information about Thing 2
- Detailed information about Thing 3
  "

File: reddwarf/database/service.py [hunk 3]
```diff
@@ -87,15 +110,19 @@ def create(self, req, body, tenant_id):
         #   code. Or maybe we shouldnt due to the nature of changing images.
         #   This needs discussion.
         # TODO(hub-cap): turn this into middleware
+        LOG.info("Creating a database instance for tenant '%s'" % tenant_id)
+        LOG.info("req : '%s'\n\n" % req)
+        LOG.info("body : '%s'\n\n" % body)
+        params = self._extract_required_params(body, 'instance')
+        LOG.info("instance params : '%s'\n\n" % params)
```
Comment (hub-cap) [inline reddwarf/database/service.py]: lets nuke these 2 lines

**Comment by hub-cap** [comment]
+1000

**Comment by TimSimpsonR** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**